README.md

```diff
@@ -21,6 +21,14 @@ git clone git@github.com:lagom/online-auction-java.git
 cd online-auction-java
 sbt runAll
 ```
+## Importing into IDEs
+
+### Intellij IDEA
+* **[Follow steps given here](http://www.lagomframework.com/documentation/1.3.x/java/IntellijSbtJava.html)**
```

Comment:
Please avoid the use of bold (aka double asterisk) if the test is already a link.

See also following links.